src/editors/data/redux/app/selectors.js

```diff
@@ -82,11 +82,25 @@ export const isRaw = createSelector(
   },
 );
 
+export const isLibrary = createSelector(
+  [module.simpleSelectors.blockId],
```

Comment:
Why are we using blockID and not learningcontextID here? Just from https://github.com/edx/frontend-lib-content-components/commit/cc4e19cd2abee9249873f5c0507a8b2791dbcdf1 we used learning contextID

Reply:
Also, we should probably have a central util which determines if something is a library so having these two paths match in what they check (`if (learningContextId && learningContextId.includes('library-v1')) {` versus ` if (blockId.startsWith('lib')) {` )Should be the same kind of check.

Reply:
I used `blockId` because I knew that was specific for a block. I didn't notice the `learningContextId`.  But I will update it to match.